decidim-debates/spec/commands/decidim/debates/update_debate_spec.rb

```diff
@@ -107,4 +113,24 @@
       expect(action_log.version.event).to eq "update"
     end
   end
+
+  context "when everything is ok with attachments" do
```

Comment:
Could you add a spec with what happens what if you upload a document that is not allowed? for example a ODT file ?